lib/internal/errors.js

```diff
@@ -287,6 +287,8 @@ E('ERR_TRANSFORM_WITH_LENGTH_0',
 E('ERR_UNESCAPED_CHARACTERS',
   (name) => `${name} contains unescaped characters`);
 E('ERR_UNKNOWN_ENCODING', 'Unknown encoding: %s');
+E('ERR_UNKNOWN_FILE_EXTENSION', 'Uknown file extension: %s');
+E('ERR_UNKNOWN_LOADER', 'Uknown module loader: %s');
```

Comment:
here too :-)

Reply:
The message here seems a bit vague - what does this mean?